pkg/server/plugin_invoker_test.go

```diff
@@ -43,139 +52,446 @@ type TestCase struct {
 var _ = Describe("PluginInvoker", func() {
```

Comment:
I'm afraid I only skimmed this testcase. I _think_ there's a missing test of a function which returns a sequence of output values and then an error. We need to make sure that all the output values are received. I'm not sure how to make such a test deterministic, but it's worth considering.

Reply:
Good stuff. Thanks.